common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -311,6 +318,34 @@ static void copyMemory(Object src, long srcOffset, Object dst, long dstOffset, l
         }
     }
 
+    static boolean equals(byte[] bytes1, byte[] bytes2) {
```

Comment:
@buchgr Yeah, I looked up the `vmSymbols.hpp` and it seems like they only do that for `char[]` and `Object[]`. Maybe they thought it's fast enough already.